Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/source/StatisticsProvider.kt

```diff
@@ -0,0 +1,102 @@
+package de.rki.coronawarnapp.statistics.source
+
+import de.rki.coronawarnapp.server.protocols.internal.stats.KeyFigureCardOuterClass
+import de.rki.coronawarnapp.statistics.IncidenceStats
+import de.rki.coronawarnapp.statistics.InfectionStats
+import de.rki.coronawarnapp.statistics.KeySubmissionsStats
+import de.rki.coronawarnapp.statistics.StatisticsData
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.filterNotNull
+import kotlinx.coroutines.launch
+import org.joda.time.Instant
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class StatisticsProvider @Inject constructor() {
+
+    private val currentInternal = MutableStateFlow(StatisticsData(items = emptyList()))
+    val current: Flow<StatisticsData> = currentInternal.filterNotNull()
+
+    init {
+        // Mock data
+        GlobalScope.launch(context = Dispatchers.IO) {
```

Comment:
it is better to inject the dispatcher

Reply:
Agree, but this is all just placeholders so multiple people can start working on this, I'll rewrite the whole class next week.

Reply:
ok